knowledge-base/actions/dsaltares/fetch-gh-release-asset/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Fetch Github Release Asset'
+github-token:
+  action-input:
+    input: token
+    is-default: true
+  permissions:
```

Comment:
There is no need for adding permissions for this KB because if there is only `content: read` permission, then there is no need of adding permission as it gets added at workflow level by default.